cmd/trade.go

```diff
@@ -188,3 +193,18 @@ func deleteAllOffersAndExit(botConfig trader.BotConfig, client *horizon.Client,
 		log.Fatal("...nothing to delete, exiting")
 	}
 }
+
+func setLogFile() {
+	t := time.Now().String()
+	var b strings.Builder
+	b.WriteString("KelpLog_")
```

Comment:
don't worry about using string builder etc. -- can just use `fileName := fmt.Sprintf("%s_%s", prefix, t)`, which should be in the calling method, based on the comment above